packages/broker/src/plugins/testnetMiner/TestnetMinerPlugin.ts

```diff
@@ -0,0 +1,132 @@
+import { Wallet } from 'ethers'
+import fetch from 'node-fetch'
+// @ts-expect-error no type definitions
+import fetchNatType from 'nat-type-identifier'
```

Comment:
Add the typing yourself (provided this is a reasonably sized library), see "packages/network/src/types/speedometer/index.d.ts" for an example.

Reply:
Fixed